src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -60,13 +84,17 @@ const ActiveState = ({
                                 translations={translations}
                             />
                         </li>
-                    );
-                case 'task':
-                    return item.task_assignment_collection.total_count ? (
-                        <li key={type + id} className="bcs-activity-feed-task">
+                    </CellMeasurer>
+                );
+
+            case 'task':
+                return item.task_assignment_collection.total_count ? (
+                    <CellMeasurer cache={this.cache} columnIndex={0} key={key} parent={parent} rowIndex={index}>
+                        <li key={type + id} className="bcs-activity-feed-task" style={style}>
                             <Task
                                 {...item}
                                 currentUser={currentUser}
+                                cellMeasurerCache={this.cache}
```

Comment:
Does Task need the cache?

Reply:
nope, I'm about to push a change removing this. I was working on an internal action changing the height of a row.

Reply:
I'm actively working on the Task area so hopefully we don't have conflicts

Reply:
I'm currently adding a prop, but otherwise the underlying components will not be changing too much.